src/ContactForm.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+
+class ContactForm extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      name: '',
+      avatar: '',
+      occupation: '',
+    }
+  }
+
+  handleNameChange(event) {
```

Comment:
See if you can reduce the three `handle` functions into one, there is a blurb in the lesson about it!